### PR TITLE
Docs: How to contribute to this package

### PR DESCRIPTION
**Pull request by literat**
Simple docs about the process of publishing this package.

Please add any suggestions to make this better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+[![Code Quality Checks][ccc-badge]][ccc-action]
+[![@lmc-eu/spirit-design-tokens][npm-badge]][npm-pkg]
+
+Design tokens for Jobs.cz brand. Based on [Spirit Design Tokens][spirit-dt]. Automatically generated from [Figma][figma] using [Supernova][supernova].
+
+## Contributing
+
+Changes from [Figma][figma] are imported and parsed using [Supernova][supernova], which is generating pull-requests with those updates for us.
+
+Version bumping is done using semantic versioning. Version type is determined by pull-request labels. Please use `BC` for major changes, `feature` for minor changes and `bug` for patches.
```
Comment (crishpeen) [inline README.md:12]:
```suggestion
Version bumping is done using semantic versioning. The version type is determined by pull-request labels. Please use `BC` for major changes, `feature` for minor changes, and `bug` for patches.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+
+Package publishing is done automatically using Github Actions by merging the pull-request into main branch.
```
Comment (crishpeen) [inline README.md:14]:
```suggestion
Package publishing is done automatically using Github Actions by merging the pull-request into the main branch.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
+# Jobs Design Tokens
+
+[![Code Quality Checks][ccc-badge]][ccc-action]
+[![@lmc-eu/spirit-design-tokens][npm-badge]][npm-pkg]
+
+Design tokens for Jobs.cz brand. Based on [Spirit Design Tokens][spirit-dt]. Automatically generated from [Figma][figma] using [Supernova][supernova].
+
+## Contributing
+
+Changes from [Figma][figma] are imported and parsed using [Supernova][supernova], which is generating pull-requests with those updates for us.
```
Comment (crishpeen) [inline README.md:10]: Maybe we can link the exporter here?

**Review by crishpeen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (literat) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (literat) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (literat) [reply README.md]: Added.

**Review by crishpeen — APPROVED** [review]

**Pull request merged**